testnets/celestiatestnet2/chain.json

```diff
@@ -31,49 +31,54 @@
     "versions": [
       {
         "name": "v1.3.0",
-        "recommended_version": "v1.3.0",
-        "compatible_versions": ["v1.3.0"]
+        "recommended_version": "v1.6.0",
+        "compatible_versions": ["v1.3.0", "v1.4.0", "v1.5.0", "v1.6.0"]
```

Comment:
👍 yep these are all compatible

hmm the JSON in the `versions` block seems duplicative with the versions one object level up. Do we also want to change `codebase.recommended_version` and `codebase.compatible_versions`?

Also I think we want to update the genesis_url to https://raw.githubusercontent.com/celestiaorg/networks/master/arabica-11/genesis.json

Reply:
> Do we also want to change codebase.recommended_version and codebase.compatible_versions?
Yes

Reply:
d9d8b665ae6f9b846f4e5ac23c5d02f4af92e13a done both here